### PR TITLE
Removing Old Hub

### PR DESCRIPTION
**Pull request by JuliusKarl**
Removing old ResearchHub code.

### CODE REVIEW

File: cer-graphql/index.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ async function createServer(config) {
                 var requestedFields = []; // List of fields requested, populated in the recursive function below
 
                 let recursive_iteration_count = 0; // Tracks the number of recursive iterations
-                const MAX_RECURSIVE_ITERATIONS = 500; // The maximum allowed number of recursive iterations
+                const MAX_RECURSIVE_ITERATIONS = 2000; // The maximum allowed number of recursive iterations
```
Comment (Hganavak) [inline cer-graphql/index.js:135]: How dare you update my arbitrary number with another arbitrary number :O

File: research-hub-web/src/app/app.component.html [hunk 2]
```diff
@@ -271,44 +266,41 @@ <h4>{{ summary }}</h4>
...
                   ngClass.lt-sm="logo-img-xs"
-                  [src]="apiService.getAssetUrl('page-elements/uoa-logo.svg')">
+                  src="assets/images/uoa-logo.svg">
               </a>
             </div>
 
-            <div fxLayout="row" fxLayout.xs="column">
+            <div fxLayout="row" fxLayout.lt-sm="column">
               <a routerLink="/userStudy">Join User Study</a>
-              <span fxHide.xs class="footer-divider">•</span>
```
Comment (Hganavak) [inline research-hub-web/src/app/app.component.html:295]: How about:

```css
.foot-divider::before {
  content: "•";
}
```

File: research-hub-web/src/styles.scss [hunk 3]
```diff
@@ -335,3 +335,38 @@ svg {
   transform: translate(1.25vh);
   transition: transform 0.2s ease-out
 }
+
+
+/**************************/
+
+// Size/Media Query Mixins /
+
+/**************************/
```
Comment (Hganavak) [inline research-hub-web/src/styles.scss:344]: Wow, excellent idea haha 👍

**Comment by Hganavak** [comment]
Some really professional looking code in there @JuliusKarl , great job 👏

**Review by rosemcc — COMMENTED** [review]
Hey Julius, all looks fine, just a couple of things: 
- Just wondering if you would want to keep the search results / search filters components/ services seeing we may want to reuse them somewhat. Or do you plan to just rebuild the search from scratch (might be easier??)? 
- Noticed the issue where there is a small white border at the bottom of the page has appeared again.. but maybe that is addressed in a different PR? Seems to appear when the page is loading when you click any of the Browse Categories buttons.

**Comment by rosemcc** [comment]
Hi Sam

**Comment by JuliusKarl** [comment]
Update
- I'm considering rebuilding the search and then incorporating old logic (_where/if_) necessary
- Fixed the footer problem
- I want to merge the front-page pr soon after this so there might be some scalability bugs with the current front-page (_resolved in front-page pr_)

File: cer-graphql/index.js (hunk 1 above)
Comment (JuliusKarl) [reply cer-graphql/index.js:135]: its bigger and better

File: research-hub-web/src/app/app.component.html (hunk 2 above)
Comment (JuliusKarl) [reply research-hub-web/src/app/app.component.html:295]: I just removed it entirely

**Comment by rosemcc** [comment]
Cool, thanks Julius, feel free to merge

**Pull request merged**